jupyterlab/packages/jupyterlab-jupytext-extension/README.md

```diff
@@ -22,16 +22,30 @@ jupyter labextension install jupyterlab-jupytext@1.1.1  # for JupyterLab 1.x
 
 # How to develop this extension
 
-We assume that you have activated the conda environment described in [CONTRIBUTING.md](https://github.com/mwouts/jupytext/blob/main/CONTRIBUTING.md).
+For fine-grained access to the `jlpm` command and various build steps:
 
-Then you can rebuild the Jupytext python package (with `python -m build`) and reinstall it (`pip install dist/jupytext-x.x.x-py3-none-any.whl`).
+```bash
+pip install -e '.[dev]'
+cd jupyterlab/packages/jupyterlab-jupytext
+jlpm
+jlpm install:extension     # Symlink into `{sys.prefix}/share/jupyter/labextensions`
+```
 
-Alternatively, if you prefer to develop iteratively, you could install a development version of the extension with
+Watch the source directory and automatically rebuild the `lib` folder:
 
 ```bash
-jupyter labextension develop . --overwrite
+cd jupyterlab/packages/jupyterlab-jupytext
+# Watch the source directory in one terminal, automatically rebuilding when needed
+jlpm watch
+# Run JupyterLab in another terminal
+jupyter lab
 ```
 
+While running `jlpm watch`, every saved change to a `.ts` file will immediately be
```

Comment:
This is very helpful! Thanks!

Reply:
about `jlpm watch` it is indeed very convenient, just keep in mind the `.ts` part, because it won't trigger if you change a `.json` or a `.css` though, so in these cases it is important to kill it and restart it, at least that was my experience